code/controllers/stonedmc/admin.dm

```diff
@@ -0,0 +1,53 @@
+// Clickable stat() button.
+/obj/effect/statclick
+	name = "Initializing..."
+	var/target
+
+INITIALIZE_IMMEDIATE(/obj/effect/statclick)
+
+/obj/effect/statclick/Initialize(mapload, text, target) //Don't port this to Initialize it's too critical
```

Comment:
```suggestion
/obj/effect/statclick/Initialize(mapload, text, targ) //Don't port this to Initialize it's too critical
```

Reply:
This is one of the few times `src.` is allowed, this pattern is acceptable.